packages/devtools/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@friggframework/devtools",
+  "prettier": "@friggframework/prettier-config",
+  "version": "1.0.0",
+  "dependencies": {
+    "@friggframework/core": "^1.0.0",
```

Comment:
This may be semantic, but in theory couldn't all of these be devDependencies?

Reply:
yeah I had that thought too, wasn't sure what to make of that. they could also all be just regular dependencies since the dev is implied